### PR TITLE
feat(DDS): dds supports backup policy and second level monitor updated

### PR DESCRIPTION
**Pull request by 346010762**
**What this PR does / why we need it**:
dds supports backup policy and second level monitor updated

**Special notes for your reviewer**:

**Release note**:

```release-note
dds supports backup policy and second level monitor updated
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/dds/ TESTARGS='-run TestAccDDSV3Instance'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dds/ -v -run TestAccDDSV3Instance -timeout 360m -parallel 4
=== RUN   TestAccDDSV3Instance_basic
=== PAUSE TestAccDDSV3Instance_basic
=== RUN   TestAccDDSV3Instance_withEpsId
=== PAUSE TestAccDDSV3Instance_withEpsId
=== RUN   TestAccDDSV3Instance_prePaid
=== PAUSE TestAccDDSV3Instance_prePaid
=== RUN   TestAccDDSV3Instance_withConfigurationSharding
=== PAUSE TestAccDDSV3Instance_withConfigurationSharding
=== RUN   TestAccDDSV3Instance_withConfigurationReplicaSet
=== PAUSE TestAccDDSV3Instance_withConfigurationReplicaSet
=== RUN   TestAccDDSV3Instance_withSecondLevelMonitoring
=== PAUSE TestAccDDSV3Instance_withSecondLevelMonitoring
=== CONT  TestAccDDSV3Instance_basic
=== CONT  TestAccDDSV3Instance_withConfigurationSharding
=== CONT  TestAccDDSV3Instance_prePaid
=== CONT  TestAccDDSV3Instance_withEpsId
--- PASS: TestAccDDSV3Instance_withConfigurationSharding (862.25s)
=== CONT  TestAccDDSV3Instance_withSecondLevelMonitoring
--- PASS: TestAccDDSV3Instance_withSecondLevelMonitoring (831.67s)
=== CONT  TestAccDDSV3Instance_withConfigurationReplicaSet
--- PASS: TestAccDDSV3Instance_prePaid (1977.87s)
--- PASS: TestAccDDSV3Instance_withConfigurationReplicaSet (893.63s)
--- PASS: TestAccDDSV3Instance_withEpsId (3011.31s)
--- PASS: TestAccDDSV3Instance_basic (3016.79s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dds       3016.834s

```

### CODE REVIEW

File: docs/resources/dds_instance.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ The following arguments are supported:
 
 **NOTE:** The instance will be restarted in the background when switching SSL. Please operate with caution.
 
+* `enabled` - (Optional, Bool) Specifies whether to enable second level monitoring.
```
Comment (Jason-Zhang9309) [inline docs/resources/dds_instance.md]: this parameter name should be change, we usually use it to indicate whether the instance is enabled

File: docs/resources/dds_instance.md [hunk 2]
```diff
@@ -235,17 +237,24 @@ The `flavor` block supports:
...
+
+* `period` - (Optional, String) Specifies the backup cycle configuration. Data will be automatically backed up on the
+  selected days every week.
+  Value range: The value is a list of digits separated by commas (,). Each digit indicates a day of the week. The
+  restrictions on the backup retention period are as follows:
+  + This parameter is no need if the retention period is set to 0.
+  + If you set the retention period to 1 to 6 days, data is automatically backed up each day of the week. Set the
+    parameter value to 1,2,3,4,5,6,7.
+  + If you set the retention period to 7 to 732 days, select at least one day of the week for the backup cycle. Example
+    value: 1,2,3,4.
```
Comment (Jason-Zhang9309) [inline docs/resources/dds_instance.md]: this description is confusing, please make it more clear

File: docs/resources/dds_instance.md [hunk 3]
```diff
@@ -235,17 +237,24 @@ The `flavor` block supports:
...
-  + If this parameter is not transferred, the automated backup policy is enabled by default. Backup files are stored
-      for seven days by default.
+  from 0 to 732. If this parameter is set to 0, the automated backup policy is disabled.
+
+* `period` - (Optional, String) Specifies the backup cycle configuration. Data will be automatically backed up on the
+  selected days every week.
+  Value range: The value is a list of digits separated by commas (,). Each digit indicates a day of the week. The
+  restrictions on the backup retention period are as follows:
+  + This parameter is no need if the retention period is set to 0.
+  + If you set the retention period to 1 to 6 days, data is automatically backed up each day of the week. Set the
```
Comment (Jason-Zhang9309) [inline docs/resources/dds_instance.md]:
```suggestion
  + If you set the `keep_days` to 1 to 6 days, data is automatically backed up each day of the week. Set the
```

File: docs/resources/dds_instance.md (hunk 1 above)
Comment (346010762) [reply docs/resources/dds_instance.md]: second_level_monitoring_enabled

File: docs/resources/dds_instance.md (hunk 2 above)
Comment (346010762) [reply docs/resources/dds_instance.md]: done

File: docs/resources/dds_instance.md (hunk 3 above)
Comment (346010762) [reply docs/resources/dds_instance.md]: done

**Pull request merged**